docs/guide/data.md

```diff
@@ -243,15 +243,15 @@ export default {
   },
 
   // Client-side only
-  mounted () {
-    // We already incremented 'count' on the server
-    // We know by checking if the 'foo' state already exists
-    const alreadyIncremented = !!this.$store.state.foo
+  beforeMount () {
+    // We have to check if there is no data
+    const storeCountEmpty = !!this.$store.state.foo.count
```

Comment:
I'm not sure this line should be changed. `!!this.$store.state.foo` already checks if the state of the dynamic module is here.